lib/src/codecs/readonly_date.ts

```diff
@@ -21,3 +21,20 @@ export const readOnlyDateFromISOString = ITT.DateFromISOString.pipe(
   readonlyDateFromDate,
   "ReadonlyDate from ISO string"
 );
+
+export const ReadonlyDateC = new T.Type<ReadonlyDate, Date, Date>(
+  "ReadonlyDate",
+  ITT.date.is,
+  (u: Date) => E.of(readonlyDate(u)),
+  (a) => new Date(a.valueOf())
+);
+
+/**
+ * Codec to convert between a readonly and a readonly date.
```

Comment:
`...between a Date and a ReadonlyDate` ?

Reply:
Sentence changed to "Identity codec to convert between a readonly date and a readonly date. Will work if input is either date or readonly date."